pages/article/[articleLink].js

```diff
@@ -17,14 +18,17 @@ const Article = () => {
 
   const [article, setArticle] = useState();
   const [isDetailsOpen, setIsDetailsOpen] = useState(false);
+  const [user, setUser] = useState();
 
-  const openDetailsHandler = () => {
+  const openDetails = () => {
     setIsDetailsOpen(!isDetailsOpen);
   };
 
   useEffect(() => {
     const handleFetching = async () => {
-      setArticle(await getArticleDetail(articleLink));
+      const data = await getArticleDetail(articleLink);
+      setArticle(data);
+      setUser(await getUserDetail(data.author));
```

Comment:
Why do we have two conventions on how to handle the async/await next to each other? Is there something I am missing?

We should probably stick with the nested one, since you have already committed to that in all of the places